version/version.go

```diff
@@ -0,0 +1,37 @@
+package version
+
+import (
+	"github.com/lyft/flyteidl/gen/pb-go/flyteidl/admin"
+	"time"
+
+	"github.com/sirupsen/logrus"
+)
+
+// This module provides the ability to inject Build (git sha) and Version information at compile time.
```

Comment:
can we not re-use directly from flytestdlib - it is a dependency

Reply:
This version package is for flytectl. And we are setting the value through ldflags

Reply:
You should be able to use stdlib.. all it has is a copy of this file... you can set it there in LDFlags, then just read it from the same file when the version command is invoked... It'll not be setting the version of stdlib or anything... it's still just setting the version in this binary...